examples/playground-movies/helpers.ts

```diff
@@ -6,7 +6,7 @@ import OpenAI from "openai";
 init({
   apiKey: process.env.GENTRACE_API_KEY ?? "",
   runName: "Example run for Playground SDK",
-  basePath: "http://localhost:3000/api",
+  basePath: process.env.GENTRACE_BASE_PATH ?? "",
```

Comment:
We normally don't tell users to configure `basePath`, so `GENTRACE_BASE_PATH` might be empty a lot. Does it fall back to prod if `basePath` == ""?

Reply:
yes, that's what i saw